it/integration_test.go

```diff
@@ -86,9 +87,18 @@ func TestMigrator__integration(t *testing.T) {
 
 	// write values
 	for i := range data {
-		client.Write(data[i].path, map[string]string{
-			data[i].key: data[i].value,
-		}, "1")
+		m := make(map[string]interface{})
+
+		switch d := data[i].value.(type) {
+		case int:
+			m[data[i].key] = d
+		case string:
+			m[data[i].key] = d
+		default:
+			t.Fatal("Error: unsupported data type")
+		}
+		client.Write(data[i].path, m, "1")
+
 		kv := client.Read(data[i].path)
 		if kv[data[i].key] != data[i].value {
```

Comment:
Is `reflect.DeepEqual` easier to use here?

Reply:
yes, I think the same, json unmarshal code or reflection should be here, just wanted to finish this one really quick, but stuck on integration test